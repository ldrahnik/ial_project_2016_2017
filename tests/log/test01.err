Negative cycle detection which bellman ford can not handle.
