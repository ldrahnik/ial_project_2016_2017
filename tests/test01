A 2 D 2 B A 5 B 1 C 7 E 10 F C 3 F C 2 D E 3 D
