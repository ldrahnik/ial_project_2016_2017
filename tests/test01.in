A -2 D
D 1 A
A 5 B
B 1 C
C 7 E
C 3 F
C 2 D
D 2 B
E 10 F 
E 3 D
