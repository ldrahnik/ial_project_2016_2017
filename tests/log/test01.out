
Floyd Warshall:
-> A -> D -> B -> C -> F 

Bellman Ford:
